tests/BUILD

```diff
@@ -35,6 +35,17 @@ cc_test(
     ],
 )
 
+cc_library(
+    name = "gates_cirq_testfixture",
```

Comment:
_[No action required]_ On reading this, I initially thought that `gates_cirq` should have its own dedicated test, similar to `gates_qsim_test` below. However, `gates_qsim_test` reads like a copy of `gates_qsim` - I don't think it provides much in the way of coverage beyond what `run_qsim[h]_test` offers.

Reply:
This is a good point. Well, `run_qsim[h]_test` don't really provide unit tests for `gates_qsim` or for `gates_cirq`. I was planning to add its own dedicated test for `gates_cirq` (similar to `gates_qsim`).